docs/detections/add-exceptions.asciidoc

```diff
@@ -0,0 +1,241 @@
+=======
+    * `matches` | `does not match` — Allows you to use wildcards in *Value*, such as `C:\path\*\app.exe`. Available wildcards are `?` (match one character) and `*` (match zero or more characters). The selected *Field* data type must be {ref}/keyword.html#keyword-field-type[keyword], {ref}/text.html#text-field-type[text], or {ref}/keyword.html#wildcard-field-type[wildcard].
++
+IMPORTANT: Using wildcards can impact performance. To create a more efficient exception using wildcards, use multiple conditions and make them as specific as possible. For example, adding conditions using `process.name` or `file.name` can help limit the scope of wildcard matching.
+
+  .. *Value*: Enter the value associated with the *Field*. To enter multiple values (when using `is one of` or `is not one of`), enter each value, then press **Return**.
+
+. Click *AND* or *OR* to create multiple conditions and define their relationships.
+
+. Click *Add nested condition* to create conditions using nested fields. This is only required for
+<<nested-field-list, these nested fields>>. For all other fields, nested conditions should not be used.
+
+. Choose to add the exception to the rule or a shared exception list. 
++
+NOTE: If you are creating an exception from the Rule Exceptions page, you can add the exception to multiple rules.  
++ 
+TIP: If a shared exception list doesn't exist, you can <<shared-exception-lists,create one>> from the Rule Exceptions page.
+
+. (Optional) Enter a comment describing the exception.
+
+. Select one of the following alert actions:
+
+* *Close this alert*: Closes the alert when the exception is added. This option
+is only available when adding exceptions from the Alerts table.
+* *Close all alerts that match this exception and were generated by this rule*:
+Closes all alerts that match the exception's conditions and were generated only by the current rule.
++
+. Click *Add rule exception*. 
+
+[float]
+[[endpoint-rule-exceptions]]
+=== Add {elastic-endpoint} exceptions
+
+Like detection rule exceptions, you can add Endpoint agent exceptions either by editing the Endpoint Security rule or by adding them as an action on alerts generated by the Endpoint Security rule. {elastic-endpoint} alerts have the following fields:
+
+* `kibana.alert.original_event.module determined:endpoint`
+* `kibana.alert.original_event.kind:alert`
+
+You can also add Endpoint exceptions to rules that are associated with {elastic-endpoint} rule exceptions. To associate rules, when creating or editing a rule, select the <<rule-ui-advanced-params, *{elastic-endpoint} exceptions*>> option.
+
+Endpoint exceptions are added to
+both the detection rule *and* the {elastic-endpoint} agent on your hosts.
+
+[IMPORTANT]
+=============
+Exceptions added to the Endpoint Security rule affect all alerts sent
+from the Endpoint agent. Be careful not to unintentionally prevent some Endpoint
```

Comment:
```suggestion
from the Endpoint agent. Be careful not to unintentionally prevent useful
```

Reply:
Hm, I'm split on this. I kind of want to keep "Endpoint" in the second sentence because I think the repetition helps emphasize the practice we're recommending. How about:

```suggestion
from the Endpoint agent. Be careful not to unintentionally prevent useful Endpoint
```

Reply:
Hm, I'm split on this. I kind of want to keep "Endpoint" in the second sentence because I think the repetition helps emphasize the practice we're recommending. How about:

```suggestion
from the Endpoint agent. Be careful not to unintentionally prevent useful Endpoint
```

Reply:
Love it